department-of-veterans-affairs/va.gov-cms
# VACMS-10111: Adds configurable heading functionality to views pagers

## Description

Patch allows for per-view pager heading configuration.

Relates to #10111 

## Testing done

This patch has been tested against core.  It is currently failing the test @ core/modules/datetime/tests/src/Functional/Views/FilterDateTest.php with the following error:

`Drupal\Core\Config\Schema\SchemaIncompleteException: Schema errors for views.view.test_filter_datetime with the following errors: views.view.test_filter_datetime:display.default.display_options.pager.options.pagination_heading_level missing schema in Drupal\Core\Config\Development\ConfigSchemaChecker->onConfigSave() (line 94 of core/lib/Drupal/Core/Config/Development/ConfigSchemaChecker.php).`

## Screenshots

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/69f3b106-c164-46d5-ac70-3d99cf7a70de)

## QA steps

What needs to be checked to prove this works?

- Go to https://pr14841-zocaqptxdmttlxhdgak9onnf5s3didgh.ci.cms.va.gov/admin/structure/views/view/content
- Click on the Pager Options Configuration link ![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/d512add5-2ba8-4875-b6ee-2f4140d52f76)
- The _Heading Level_ configuration option should be present ![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/69f3b106-c164-46d5-ac70-3d99cf7a70de)
- Choose a heading level, click `Apply`, and then click `Save`
- Either scroll down to verify via Preview pager area, or navigate back to https://pr14841-zocaqptxdmttlxhdgak9onnf5s3didgh.ci.cms.va.gov/admin/content and locate the pager near the bottom of the page
- Right-click on or near the first link in the pager and select Inspect. The invisible header that contains the pager--its element should have been updated to reflect your selection. ![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/33521f07-2756-4e84-8182-0b8b04b6d44d)


### Select Team for PR review
